rebase.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Rebase On the Tree
+  hosts: localhost
+  become: yes
+  become_method: sudo
+
+  tasks:
+  - name: Create VM from Atomic QCOW2 image
+    command:
+      sh create-vm.sh mynode /path/to/f25-atomic.qcow2
+      # /path/to/f25-atomic.qcow2 = /var/lib/libvirt/images/Fedora-Atomic-25-20170131.0.x86_64.qcow2
+
+  - name: Get the IP of VM
+    command: virsh domifaddr mynode
+    register: VM_IP
+
+  - name: Create inventory file
+    file: path=inventory state=touch
+
+  - name: Write VM_IP in the inventory
+    lineinfile:
+      dest: inventory
+      line: '[atomichost]\n{{ VM_IP }} ansible_ssh_user=atomic-user ansible_ssh_pass=atomic'
```

Comment:
Ahh..Yeah I didn't process the output of VM_IP.
I can see that I am getting the right response, just needs processing.

```
ok: [localhost] => {
    "msg": "{u'changed': True, u'end': u'2017-02-06 18:00:23.176018', u'stdout': u' Name       MAC address          Protocol     Address\\n-------------------------------------------------------------------------------\\n vnet7      52:54:00:bc:23:0c    ipv4         192.168.121.102/24', u'cmd': [u'virsh', u'domifaddr', u'node-25'], u'start': u'2017-02-06 18:00:23.157048', u'delta': u'0:00:00.018970', u'stderr': u'', u'rc': 0, 'stdout_lines': [u' Name       MAC address          Protocol     Address', u'-------------------------------------------------------------------------------', u' vnet7      52:54:00:bc:23:0c    ipv4         192.168.121.102/24'], u'warnings': []}.stdout"
}
```